modules/rosa-sts-byo-oidc.adoc

```diff
@@ -1,29 +1,36 @@
 // Module included in the following assemblies:
 //
-// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+// * rosa_architecture/rosa-oidc-overview.adoc
 // * rosa_architecture/rosa-sts-about-iam-resources.adoc
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc
 
 ifeval::["{context}" == "rosa-hcp-sts-creating-a-cluster-quickly"]
 :rosa-hcp:
 endif::[]
-ifeval::["{context}" == "rosa-sts-about-iam-resources"]
-:rosa-classic:
-endif::[]
 
 :_content-type: PROCEDURE
-[id="rosa-hcp-byo-oidc_{context}"]
+[id="rosa-sts-byo-oidc_{context}"]
 = Creating an OpenID Connect configuration
 
-When using a {hcp-title} cluster, you must create the OpenID Connect (OIDC) configuration prior to creating your cluster. This configuration is registered to be used with OCM.
+When using a 
+ifdef::rosa-hcp[]
+{hcp-title} 
+endif::rosa-hcp[]
+ifndef::rosa-hcp[]
+{product-title} 
+endif::rosa-hcp[]
+cluster, you must create the OpenID Connect (OIDC) configuration prior to creating your cluster. This configuration is registered to be used with OCM.
```

Comment:
 For the Classic ROSA cluster, the OIDC configuration is not "must"

Reply:
@yingzhanredhat Thanks for that! I updated it to say "you may" on ROSA classic while ROSA with HCP still says "you must."